src/pnotify.animate.js

```diff
@@ -94,7 +95,7 @@
             this.notice.animating = "out";
             var that = this;
             callback = (function(){
-                that.notice.elem.removeClass("ui-pnotify-in");
+                that.notice.elem.removeClass("ui-pnotify-in").removeClass(that.options.out_class);
```

Comment:
In the case that out_class is undefined, this would remove all the classes on the element.